src/containers/CoursePeoplePage.jsx

```diff
@@ -1,32 +1,83 @@
 import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
 import PersonComponent from '../components/PersonComponent';
 import SectionLevelBar from '../components/SectionLevelBar';
 import AddUserToGroupButtons from '../components/AddUserToGroupButtons';
 import AddMentorToGroupButton from '../components/AddMentorToGroupButton';
+import { getPeopleInGroup } from '../actions/groupActions';
 
-function SessionPage() {
-  return (
-    <section className="section">
-      <div className="container">
-        <SectionLevelBar title="Mentors">
-          <AddMentorToGroupButton />
-        </SectionLevelBar>
-        <div className="columns is-multiline is-mobile bordered">
-          <PersonComponent />
-        </div>
-        <SectionLevelBar title="Mentees">
-          <AddUserToGroupButtons />
-        </SectionLevelBar>
-        <div className="columns is-multiline is-mobile bordered">
-          <PersonComponent firstName="Stephen" lastName="Supercalifgailisticexpealodocious" />
+class CoursePeoplePage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      id: '',
+    };
+  }
 
-          <PersonComponent />
+  componentDidMount() {
+    // only run once
```

Comment:
Technically this isn't correct, it runs every time the component gets mounted in the virtual DOM again. You can watch Redux debugger and see that the `GET_PEOPLE_IN_GROUP` action runs each time.

I don't mind though, because this works and shows no display glitches. 👍